reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -8934,6 +8937,17 @@ public final void subscribe(Subscriber<? super T> actual) {
 	 * @see #as for a loose conversion to an arbitrary type
 	 */
 	public final <V> Flux<V> transform(Function<? super Flux<T>, ? extends Publisher<V>> transformer) {
```

Comment:
do we need this on `transform`? keeping things on `transformDeferred` would align well with #1890 and keep the message simple and consistent: if you're using an RS library other than Reactor, scope it into `transformDeferred`.